js/researches/german/GermanParticiple.js

```diff
@@ -73,4 +78,15 @@ GermanParticiple.prototype.hasHabenSeinException = function() {
 	return includes( habenSeinIndices, currentParticiple.index + currentParticiple.match.length + 1 );
 };
 
+/**
+ * Checks whether a found participle is an auxiliary.
+ * If a word is an auxiliary, it isn't a participle.
+ *
+ * @returns {boolean} Returns true if it is an auxiliary, otherwise returns false.
+ */
+GermanParticiple.prototype.isAuxiliary = function() {
+	return this.getParticiple().match( auxiliaryRegex ) !== null;
```

Comment:
Please don't use regexes for this, creating a function like `isInExceptionList` works a bit faster.